chroma-dependencies/Makefile

```diff
@@ -21,7 +21,7 @@ cleanrepo: TARGET=cleanrepo
 requirements::
 	# these are the ones provided by EL and EPEL and our own repo
 	set -e;                                                   \
-	{ echo "../chroma-externals/netaddr-0.7.5.zip";           \
+	{ echo "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"; \
```

Comment:
So, to close on this; I think we need to update our standards on formatting enforcement in code review.

1. We should be focusing on reviewing the content of code not how long a line is and consistent continuations. There will always be exceptions to every rule and pragmatism has to win the day over rigidity if we want to move forward.
2. Having a preferred style is ok, but it should be enforced at checkin time and not code review. There exist tools that not only enforce code styles but actually fix them for you: https://github.com/prettier/prettier, https://github.com/dungpa/fantomas. Unsure of existing tooling in the makefile / bash / python space.
3. If we feel strongly enough about not passing reviews due to formatting inconsistencies, we should invest time into tooling that will parse / reformat code according to our styles. That way we get the benefits of consistent code, without the need for the code-review team to lint the code as well as review content. @tanabarr perhaps this would be something you'd be interested in working on?